web/react/components/sidebar.jsx

```diff
@@ -100,74 +100,56 @@ export default class Sidebar extends React.Component {
     }
     getStateFromStores() {
         const members = ChannelStore.getAllMembers();
-        var teamMemberMap = UserStore.getActiveOnlyProfiles();
-        var currentId = ChannelStore.getCurrentId();
-        const currentUserId = UserStore.getCurrentId();
+        const currentChannelId = ChannelStore.getCurrentId();
 
-        var teammates = [];
-        for (var id in teamMemberMap) {
-            if (id === currentUserId) {
-                continue;
-            }
-            teammates.push(teamMemberMap[id]);
-        }
+        const channels = Object.assign([], ChannelStore.getAll());
```

Comment:
`channels` is never mutated so copying it seems unnecessary. I'm also not sure if Object.assign actually works for arrays, but I could be wrong